packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -166,106 +139,9 @@ contract FederatedAttestations is
 
   /**
    * @notice Returns info about up to `maxAttestations` attestations for
-   *   `identifier` produced by unrevoked signers of `trustedIssuers`
+   *   `identifier` produced signers of `trustedIssuers`
    * @param identifier Hash of the identifier
    * @param trustedIssuers Array of n issuers whose attestations will be included
-   * @param maxAttestations Limit the number of attestations that will be returned
-   * @return [0] Array of number of attestations returned per issuer
-   * @return [1 - 3] for m (== sum([0])) found attestations, m <= maxAttestations:
-   *         [
-   *           Array of m accounts,
-   *           Array of m issuedOns,
-   *           Array of m signers
-   *         ]; index corresponds to the same attestation
-   * @dev Adds attestation info to the arrays in order of provided trustedIssuers
-   * @dev Expectation that only one attestation exists per (identifier, issuer, account)
-   */
-  // TODO reviewers: is it preferable to return an array of `trustedIssuer` indices
```

Comment:
nit: can you add this TODO back in to the lookup attestations function so we don't lose track of revisiting this?